templates/job_request/detail.html

```diff
@@ -103,33 +103,47 @@ <h1 class="text-3xl tracking-tight break-words font-bold text-slate-900 sm:text-
 
     <div class="space-y-6 md:space-y-6 lg:col-span-2">
       {% #card container=True %}
-        <div class="flex flex-col gap-y-4 text-slate-700">
-          <p>
-            This page shows the technical details of what happened when
-            the authorised researcher
-            <strong>{{ job_request.created_by.fullname }}</strong>
-            requested one or more actions to be run against real patient data
-            in the
-            {% spaceless %}
-              {% link text="project" href=job_request.workspace.project.get_absolute_url %}
-            {% endspaceless %}, within a secure environment.
-          </p>
-          <p>
-            By cross-referencing the list of jobs with the
-            pipeline section below, you can infer what
-            {% link text="security level" href="https://docs.opensafely.org/security-levels/" %}
-            various outputs were written to. Researchers can never directly
-            view outputs marked as
-            {% #code %}highly_sensitive{% /code %};
-            they can only request that code runs against them. Outputs
-            marked as
-            {% #code %}moderately_sensitive{% /code %}
-            can be viewed by an approved researcher by logging into a highly
-            secure environment. Only outputs marked as
-            {% #code %}moderately_sensitive{% /code %}
-            can be requested for release to the public, via a controlled
-            output review service.
-          </p>
+        <div class="prose prose-oxford">
+          {% if job_request.backend.name == "Test" %}
```

Comment:
thought: I'm kind of surprised there isn't an attribute on the Backend model that says whether the instance is test only, but there isn't, so we have to hardcode the name here.

That isn't a concern specifically about this PR though; it just highlights it.